test/common/TestBedChild.cpp

```diff
@@ -315,6 +314,14 @@ namespace RcclUnitTesting
 
   ErrCode TestBedChild::ExecuteCollectives()
   {
+    int rankListSize, tempRank;
```

Comment:
Change to receive a single rank value (-1 to represent all), similar to other functions here